src/components/organims/DownloadByOrganization.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { css } from 'emotion'
+import { Global } from '@emotion/core'
+
+import AppMinimal from '../layout/AppMinimal';
+import Seo from '../molecules/Seo';
+import Button from '../atoms/Button';
+
+
+const main = css`
+  background-size: 100% !important;
+  bottom: 0;
+  width: 100vw;
+  position: absolute;
+  height: 100vw;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  overflow-x: hidden;
+ `
+ 
+ const titleStyle = css`
+  font-size: 35px;
+  font-weight: bold;
+  margin-top: 35px;
+ `
+ 
+ const descriptionStyle = css`
+  margin-top: 25px;
+  max-width: 500px;
+  hyphens: auto;
+  text-align: center;
+ `
+ 
+ const logosContainer = css`
+  display: flex;
+  justify-content: center;
+  align-items: center ;
+  & > *{
+    margin-right: 35px;
+  }
+ `
+ 
+ const logoIcon = css`
+  width: 90px;
+  height: 90px !important;
+  border-radius: 90px;
+ `
+ 
+ const logoSeparator = css`
+  width: auto;
+  height: 30px
+ `
+ 
+ const buttonDownload = css`
+  display: flex;
+  justify-content: flex-between;
+  margin-top: 35px;
+ `
+ 
+ const videoOfStation = css`
+  height: auto;
+  width: 400px;
+  border-radius: 10.4px;
+  box-shadow: 10px 10px 42px -21px rgba(0,0,0,0.75);
+ `
+ 
+ const detailsStyle = css`
+  margin-top: 35px;
+  margin-bottom: 25px;
+  max-width: 500px;
+  hyphens: auto;
+  text-align: center;
+ `
+
+
+export default class DownloadByOrganization extends React.Component{
+
+  constructor(props){
+    super(props);
+    console.log(this)
+  }
+  render(){
+    const {onClickDownload, title, description, bkg_image, pictureUrl, linkbewteenorgandstationicons, stationicon, download_icon, button_text, details, illustration} = this.props;
+    return( 
+    <AppMinimal>
+      <Seo
+        title={title}
+        description={description}
+      />
+      <Global
+        styles={{
+          'body': {
+            overflowX: 'hidden'
+          }
+        }}
+      />
+      <section className={main} style={{background: `url(${bkg_image}) no-repeat bottom`}}>
```

Comment:
I don't see the logo of my organization:
<img width="1222" alt="Screenshot 2019-12-05 11 14 35" src="https://user-images.githubusercontent.com/687961/70226064-72db0400-1750-11ea-84cc-870da04cea8b.png">

Reply:
🧐 Will re-check that so

Reply:
There was an error inside the code + did found some interesting thing sometimes this happen with our logo at clearbit ->
<img width="1680" alt="Capture d’écran 2019-12-05 à 21 35 17" src="https://user-images.githubusercontent.com/7901366/70271857-85ccf300-17a7-11ea-854b-bf4c17245fb4.png">